getline.ts/src/blockchain.ts

```diff
@@ -144,10 +144,26 @@ export class Contract {
     public async mutate(methodName: string, ...params: any[]): Promise<void> {
         logger(`Mutating ${methodName}...`);
         const method: any = this.instance[methodName];
-        const hash: string = await (new Promise<string>((resolve, reject) => {
+        const gas: number = await (new Promise<number>((resolve, reject) => {
+            const fallback = 300000;
             const opts = {
                 gas: 1000000,
             };
+            method.estimateGas(...params, opts, (err: Error, cost: number) => {
+                if (err !== null) {
+                    logger(`Failed to estimate mutation gas consumption, falling back to default gas `
```

Comment:
Think this should actually be `console.error` due to severity

Reply:
Done.